[FrameworkBundle] Add missing webhook parsers

alexandre-daubois opened the pull request:
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

Detected when reviewing this documentation PR: https://github.com/symfony/symfony-docs/pull/19268

Should be merged after #53007, which brings notifier services removal conditionally

alexandre-daubois committed:
[FrameworkBundle] Add missing webhook parsers

TimoBakx commented on src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php:
```diff
@@ -2586,7 +2586,9 @@ private function registerMailerConfiguration(array $config, ContainerBuilder $co
+                MailerBridge\Mailjet\Webhook\MailjetRequestParser::class => 'mailer.webhook.request_parser.mailjet',
                 MailerBridge\Postmark\Webhook\PostmarkRequestParser::class => 'mailer.webhook.request_parser.postmark',
                 MailerBridge\Sendgrid\Webhook\SendgridRequestParser::class => 'mailer.webhook.request_parser.sendgrid',
             ];
```
Should the Notifier-Webhook service definitions be removed from the container as well if the bridges are not installed? Like in the foreach below here (line 2594 / 2596).

alexandre-daubois replied:
Indeed, thanks!

TimoBakx commented:
Nice work!

alexandre-daubois committed:
[FrameworkBundle] Fix webhook parser service removal and add notifier parser service removal

fabpot committed:
bug #53007 [FrameworkBundle] Fix webhook parser service removal and add notifier parser service removal (alexandre-daubois)

This PR was merged into the 6.3 branch.

Discussion
----------

[FrameworkBundle] Fix webhook parser service removal and add notifier parser service removal

| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

Should be merged before https://github.com/symfony/symfony/pull/53004

Commits
-------

b7a17ad5d0 [FrameworkBundle] Fix webhook parser service removal and add notifier parser service removal

fabpot committed:
Merge branch '6.3' into 6.4

* 6.3:
  [FrameworkBundle] Fix webhook parser service removal and add notifier parser service removal

fabpot commented:
Thank you @alexandre-daubois.
